swan-lake/integration-tutorials/content-based-message-routing.md

```diff
@@ -88,129 +88,139 @@ Follow the instructions given in this section to develop the service.
-    > Here, a separate function is used to initialize the clients to aid with testing. Alternatively, the `new` expression can be used directly to initialize the clients.
-    >
-    > ```ballerina
-    > final http:Client grandoakEP = new ("http://localhost:9090/grandoak/categories");
-    > final http:Client clemencyEP = new ("http://localhost:9090/clemency/categories");
-    > final http:Client pinevalleyEP = new ("http://localhost:9090/pinevalley/categories");
-    >```
+4. Generate record types corresponding to the payloads from the hospital backend services by providing samples of the expected JSON payloads.
 
-5. Introduce an [enum](https://ballerina.io/learn/by-example/enumerations/) to define the hospital IDs.
+    The payload from the hospital backend service will be a JSON object similar to the following.
 
-    ```ballerina
-    enum HospitalId {
-        GRAND_OAK = "grandoak",
-        CLEMENCY = "clemency",
-        PINE_VALLEY = "pinevalley"
-    };
+    ```json
+    {
+        "appointmentNumber": 1,
+        "doctor": {
+            "name": "thomas collins",
+            "hospital": "grand oak community hospital",
+            "category": "surgery",
+            "availability": "9.00 a.m - 11.00 a.m",
+            "fee": 7000.0
+        },
+        "patient": {
+            "name": "John Doe",
+            "dob": "1940-03-19",
+            "ssn": "234-23-525",
+            "address": "California",
+            "phone": "8770586755",
+            "email": "johndoe@gmail.com"
+        },
+        "hospital": "grand oak community hospital",
+        "confirmed": false,
+        "appointmentDate": "2023-10-02"
+    }
     ```
 
-6. Define records corresponding to the request payload and response payload.
+    ![Define record types](/learn/images/integration-tutorials/content-based-message-routing/define_record_types.gif)
+   
+    The generated record types will be as follows.
 
     ```ballerina
-    type Patient record {|
+    type Patient record {
         string name;
         string dob;
         string ssn;
         string address;
         string phone;
         string email;
-    |};
-
-    type ReservationRequest record {|
-        Patient patient;
-        string doctor;
-        HospitalId hospital_id;
-        string hospital;
-        string appointment_date;
-    |};
+    };
 
-    type Doctor record {|
+    type Doctor record {
         string name;
         string hospital;
         string category;
         string availability;
         decimal fee;
-    |};
+    };
 
-    type ReservationResponse record {|
+    type ReservationResponse record {
         int appointmentNumber;
         Doctor doctor;
         Patient patient;
         string hospital;
         boolean confirmed;
         string appointmentDate;
-    |};
+    };
+
+    type ReservationRequest record {
+        Patient patient;
+        string doctor;
+        HospitalId hospital_id;
+        string hospital;
+        string appointment_date;
+    };
```

Comment:
Missing this content in `Generating records` section

![image](https://github.com/ballerina-platform/ballerina-dev-website/assets/25485997/329a4009-3f64-4313-92ae-b9559ed2ddd6)